test/NuGet.Core.FuncTests/NuGet.Protocol.FuncTest/Utility.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using System.Xml.Linq;
+using NuGet.Common;
+
+namespace NuGet.Protocol.FuncTest
+{
+    public static class Utility
+    {
+        public static Tuple<string, string> ReadCredential(string feedName)
+        {
+            string fullPath = NuGetEnvironment.GetFolderPath(NuGetFolderPath.UserSettingsDirectory);
+            using (Stream configStream = File.OpenRead(Path.Combine(fullPath, "test.config")))
```

Comment:
can you paste full url here in method comment?
also I'd suggest naming it differently to avoid collisions, like `NuGet.Protocol.FuncTest.config`